File: template_store/frontend/sources/stores/{{namespace_ossep_lower}}.ts
/**
 * Nom du module         : {{lower_name}}.ts
 * Description           : 
 * 
 * Auteur                : {{user_name}}
 * Date de création      : {{datetime_full}}
 * Dernière modification : {{datetime_full}}
 * Version               : 1.0.0
 * Licence               : {{licence_name}}
 * 
 * Notes                 : 
 */

import { createSlice, type PayloadAction } from '@reduxjs/toolkit';

/**
 * État du store {{title_name}}.
 */
interface {{title_name}}State {
  
}

/**
 * État initial du store {{title_name}}.
 */
const initialState: {{title_name}}State = {
  
};

/**
 * Logique de l'état du store {{title_name}}.
 */
const {{lower_name}}Slice = createSlice({
  name: '{{lower_name}}',
  initialState,
  reducers: {

  }
});

/**
 * Définition des actions du store {{title_name}}.
 */
export const { } = {{lower_name}}Slice.actions;

/**
 * Définition du store {{title_name}}.
 */
export default {{lower_name}}Slice.reducer

File: template_table/backend/schemas/{{namespace_ossep_lower}}.py
"""
Nom du module         : {{lower_name}}.py
Description           : 

Auteur                : {{user_name}}
Date de création      : {{datetime_full}}
Dernière modification : {{datetime_full}}
Version               : 1.0.0
Licence               : {{licence_name}}

Notes                 : 
"""

from libraries.ormbase import OrmBase
from sqlalchemy.orm import Mapped, mapped_column, relationship
from sqlalchemy import ForeignKey

class {{title_name}}(OrmBase):
  """Schéma de la table {{title_name}}."""
  
  __tablename__: str = "{{lower_name}}"
  
  id: Mapped[int] = mapped_column(primary_key=True)